server/src/test/java/org/elasticsearch/repositories/RepositoriesServiceTests.java

```diff
@@ -283,18 +314,11 @@ public void testRegisterRepositorySuccessAfterCreationFailed() {
         // 2. repository creation successfully when current node become master node and repository is put again
         var request = new PutRepositoryRequest().name(repoName).type(TestRepository.TYPE);
 
-        repositoriesService.registerRepository(request, new ActionListener<>() {
-            @Override
-            public void onResponse(AcknowledgedResponse acknowledgedResponse) {
-                assertTrue(acknowledgedResponse.isAcknowledged());
-                assertThat(repositoriesService.repository(repoName), isA(TestRepository.class));
-            }
-
-            @Override
-            public void onFailure(Exception e) {
-                assert false : e;
-            }
-        });
+        var resultListener = new SubscribableListener<AcknowledgedResponse>();
+        repositoriesService.registerRepository(request, resultListener);
+        var resp = safeAwait(resultListener);
+        assertTrue(resp.isAcknowledged());
```

Comment:
nit: would rather not use abbrs. for var. names wherever poss. :)

```suggestion
        var response = safeAwait(resultListener);
        assertTrue(response.isAcknowledged());
```

Reply:
ofc